cli/internal/client/client.go

```
@@ -0,0 +1,230 @@
+	if err != nil {
+		return err
+	}
+	if afterEventID > 0 {
+		request.Header.Set("Last-Event-ID", strconv.FormatInt(afterEventID, 10))
+	}
+
+	response, err := c.httpClient.Do(request)
+	if err != nil {
+		return fmt.Errorf("open event stream: %w", err)
+	}
+	defer response.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `response.Body.Close` is not checked

(errcheck)